h/static/scripts/tests/controllers/autosuggest-dropdown-controller-test.js

```diff
@@ -0,0 +1,413 @@
+'use strict';
+
+const syn = require('syn');
+const unroll = require('../util').unroll;
+
+
+const AutosuggestDropdownController = require('../../controllers/autosuggest-dropdown-controller');
+
+
+// syn's move functionality does not fire
+// mouseenter and mouseleave events.
+const mouseMove = (()=>{
+
+  let _lastMovePos;
+
+  return (posObj) => {
+
+    if(_lastMovePos){
+      const prevEl = document.elementFromPoint(_lastMovePos.pageX, _lastMovePos.pageY);
+      const leaveEvent = document.createEvent( 'Events' );
```

Comment:
PhantomJS 2.x supports the Event constructor, so you can do `new Event('type', options)` rather than using createEvent + initEvent which I think is more readable.